gamemode/shared.lua

```diff
@@ -248,6 +249,10 @@ function GM:Initialize()
     end
 end
 
+function GM:GetSetting(setting)
+    return GetConVar("lambda_" .. setting)
```

Comment:
This needs to read the gametype.Settings table and return the correct value based on type.